src/pages/iv-rebuildInventory/form/IvRebuild.js

```diff
@@ -0,0 +1,125 @@
+import { Grid } from '@mui/material'
+import * as yup from 'yup'
+import { useContext } from 'react'
+import FormShell from 'src/components/Shared/FormShell'
+import toast from 'react-hot-toast'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { useForm } from 'src/hooks/form'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { ControlContext } from 'src/providers/ControlContext'
+import CustomDatePicker from 'src/components/Inputs/CustomDatePicker'
+import { ResourceLookup } from 'src/components/Shared/ResourceLookup'
+import { InventoryRepository } from 'src/repositories/InventoryRepository'
+
+export default function RebuildForm({ _labels, access }) {
+  const { postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+
+  const { formik } = useForm({
+    initialValues: { year: '', itemId: 0, itemName: '', date: new Date(), recordId: 'N/A' },
```

Comment:
1- It's allowed to set recordId to N/A hard coded?! keep it empty string or null
2- Remove itemName

Reply:
in case of rebuild we set the record N/A